packages/builder/package.json

```diff
@@ -13,11 +13,12 @@
     "cy:setup:ci": "node ./cypress/setup.js",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "cy:run:ci": "xvfb-run cypress run --headed --browser chrome",
+    "cy:run:ci": "cypress run --headed --browser chrome --spec  cypress/integration/createApp.spec.js",
```

Comment:
The `createApp.spec.js` is just temporary here - will remove after I'm done testing on `develop`